src/routers/posts.router.js

```diff
@@ -1,6 +1,84 @@
 const express = require('express');
+const Post = require('../entities/post.entity');
+const { createPost, readPosts, updatePost, deletePost } = require('../repository/posts.repository');
+const PostDTO = require('../dtos/postDTO.dto');
+const { route } = require('./index.router');
 const router = express.Router();
 
 //CREATE YOUR REST API UNDER HERE
+// Create A Post
+router.post('/post', async (req, res) => {
+    const postDTO = req.body;
+    try {
+        var post = new Post(
+            null,
+            postDTO.user_id,
+            postDTO.description,
+            postDTO.services,
+            postDTO.cleanliness,
+            postDTO.taste,
+            postDTO.price,
+        );
+        const result = await createPost(post);
+        res.status(200).send(`Post with id ${result.insertId} had saved successfully`);
```

Comment:
please use `res.status(201)` instead of 200

ref: https://httpstatuses.com/201#:~:text=The%20request%20has%20been%20fulfilled,by%20the%20effective%20request%20URI.